src/libponyrt/mem/pagemap.c

```diff
@@ -56,19 +56,20 @@ static const pagemap_level_t level[PAGEMAP_LEVELS] =
     POOL_INDEX((1 << L1_MASK) * sizeof(void*)) }
 };
 
-ATOMIC_TYPE(static void**) root;
+static ATOMIC_TYPE(void**) root;
 
 void* ponyint_pagemap_get(const void* m)
 {
-  void** v = atomic_load_explicit(&root, memory_order_acquire);
+  void** v = atomic_load_explicit(&root, memory_order_relaxed);
 
   for(int i = 0; i < PAGEMAP_LEVELS; i++)
   {
     if(v == NULL)
       return NULL;
 
     uintptr_t ix = ((uintptr_t)m >> level[i].shift) & level[i].mask;
-    v = (void**)v[ix];
+    ATOMIC_TYPE(void**)* av = (ATOMIC_TYPE(void**)*)&(v[ix]);
+    v = atomic_load_explicit(av, memory_order_relaxed);
```

Comment:
Is this change just about making the atomic load more explicit?

Is the "relaxed" order intentional here?